### PR TITLE
[action][spm] add `--very-verbose` option

### PR DESCRIPTION
**Pull request by att55**
### Checklist
- [x] I've run `bundle exec rspec` from the root directory to see all new and existing tests pass
- [x] I've followed the _fastlane_ code style and run `bundle exec rubocop -a` to ensure the code style is valid
- [x] I see several green `ci/circleci` builds in the "All checks have passed" section of my PR ([connect CircleCI to GitHub](https://support.circleci.com/hc/en-us/articles/360008097173-Why-aren-t-pull-requests-triggering-jobs-on-my-organization-) if not)
- [x] I've read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I've updated the documentation if necessary.

### Description

Hi there!

I found SPM has very-verbose option.

```
--very-verbose, --vv    Increase verbosity to include debug output
```

I also added to the option to spm action.

### CODE REVIEW

**Comment by att55** [comment]
I didn't add a short option because the option accepts only string length 1.
https://github.com/fastlane/fastlane/blob/286aa9be631a958f90597e548274fa072279984f/fastlane_core/lib/fastlane_core/configuration/config_item.rb#L21-L22

Should I add `--vv` after changed the length of option from 1 to 2 ?

**Comment by mollyIV** [comment]
Hello @att55 👋 

Thanks for the contribution 💯 

> Should I add --vv after changing the length of option from 1 to 2 ?

According to the [`config_item`](https://github.com/fastlane/fastlane/blob/master/fastlane_core/lib/fastlane_core/configuration/config_item.rb) codebase, [we cannot pass more than 1 character as a shortcut](https://github.com/fastlane/fastlane/blob/master/fastlane_core/lib/fastlane_core/configuration/config_item.rb#L123-L126) 👇 

```ruby
if short_option
        UI.user_error!("short_option for key :#{key} must of type String") unless short_option.kind_of?(String)
        UI.user_error!("short_option for key :#{key} must be a string of length 1") unless short_option.delete('-').length == 1
end
```

and to be honest, I'd stick to it 😉 

If we'd like to support a short option, maybe we could just use `-V` (upper-cased V). 

`short_option` requires being a `String`, which is case-sensitive 👌 

There are existing actions that leverage the same letter in upper and lower cased form, for example [pilot](https://github.com/fastlane/fastlane/blob/master/pilot/lib/pilot/options.rb)'s `-d` and `-D`.

File: fastlane/lib/fastlane/actions/spm.rb [hunk 1]
```diff
@@ -10,6 +10,7 @@ def self.run(params)
         cmd << "--configuration #{params[:configuration]}" if params[:configuration]
         cmd << "--disable-sandbox" if params[:disable_sandbox]
         cmd << "--verbose" if params[:verbose]
+        cmd << "--very-verbose" if params[:very_verbose]
```
Comment (mollyIV) [inline fastlane/lib/fastlane/actions/spm.rb:13]: Can we please add a unit test for it?  
You can take an inspiration from an existing tests in `spm_spec` for `--vebose` flag 💪

**Review by lacostej — APPROVED** [review]
Looks good to me. Bonus point for the unit test if you add i.t

File: fastlane/lib/fastlane/actions/spm.rb (hunk 1 above)
Comment (att55) [reply fastlane/lib/fastlane/actions/spm.rb:13]: Sorry for late reply 🙏🏼 
I will add a unit test!

**Review by rogerluan — COMMENTED** [review]
This PR also LGTM 😊 just missing the unit tests really @att55 🙇 

Also, there are merge conflicts that need to be solved before we can merge this 🙏 

Thank you!

**Comment by att55** [comment]
@rogerluan @mollyIV 
I'm so sorry for late reply. 😢
I added unit tests! Could you please check this? 🙏🏼

**Review by lucgrabowski — APPROVED** [review]
LGTM :)

**Pull request merged**